.travis/install.sh

```diff
@@ -18,7 +18,6 @@ then
   if [ "$TEST_FORMAT" == 1 ]
   then
     sudo apt-get -y install clang-format-3.8
-
```

Comment:
This line is to visually separate the install step from the 'hack around travis limitations' step. Could you put it back?

(If we were to remove it, then we should also be removing similar section separating lines further down.)